packages/provider/src/TiptapCollabProvider.ts

```diff
@@ -281,7 +292,7 @@ export class TiptapCollabProvider extends HocuspocusProvider {
     return updatedThread
   }
 
-  deleteComment(threadId: TCollabThread['id'], commentId: TCollabComment['id']) {
+  deleteComment(threadId: TCollabThread['id'], commentId: TCollabComment['id'], deleteThread?: boolean) {
```

Comment:
Is `deleteThread` appropriate here, maybe this should be configured at the provider options level?

Reply:
Added an option to control this globally on the TiptapCollabProvider